__tests__/containers/AllHolidays.js

```diff
@@ -3,8 +3,8 @@ import { shallow } from 'enzyme';
 import { AllHolidays } from '../../src/components';
 import allHolidaysContainer from '../../src/components/AllHolidays/container';
 
-jest.mock('../../src/services/holidayService');
-import * as holidayService from '../../src/services/holidayService';
+jest.mock('../../src/services/eventService');
+import * as holidayService from '../../src/services/eventService';
```

Comment:
Should import name not be changed also to reflect the eventService its importing?

Reply:
Yeah. This was caught by a find and replace. Wouldn't worry about tests though because of how out of date they are. Changed regardless.